README.md

```diff
@@ -266,6 +266,7 @@ also see [comparison to similar software](./docs/versus.md)
     * ☑ realtime streaming of growing files (logfiles and such)
   * ☑ [thumbnails](#thumbnails)
     * ☑ ...of images using Pillow, pyvips, or FFmpeg
+    * ☑ ...of RAW files using rawpy
```

Comment:
Suggestion:  `of RAW images using rawpy`

Reply:
done